bin/compile

```diff
@@ -384,6 +384,11 @@ case $TOOL in
     ;;
 esac
 
+if [ ! -z ${name+x} ];
+then
+  echo -e "---\ndefault_process_types:\n  web: $(basename $name)" >> /tmp/buildpack-release-step.yml
```

Comment:
The assumption that I was making is that a user would not be pushing an app without a `Procfile` unless that app is a simple/smaller app that would result in one binary (from the main package) being generated, with the intention that this generated binary would be run for the app. If the app was more complex and would result in multiple binaries being generated, then the user would include a `Procfile` to start different worker types off of the different binaries. Are there inaccuracies with this assumption?